Sources/ApolloTestSupport/MockOperation.swift

```diff
@@ -5,12 +5,8 @@ open class MockOperation<SelectionSet: RootSelectionSet>: GraphQLOperation {
 
   public let operationType: GraphQLOperationType
 
-  public var operationDefinition: String = "None"
-  public var operationIdentifier: String?
   public var operationName: String = "MockOperationName"
-
-  public var stubbedQueryDocument: String?
-  public final var queryDocument: String { stubbedQueryDocument ?? operationDefinition }
+  public var document: DocumentType = .notPersisted(definition: .init("None"))
```

Comment:
I would make this something like "Mock" rather than preserving the existing "None" for clarity

Reply:
Fair enough. Will do!